### PR TITLE
Testing changes

### PR DESCRIPTION
**Pull request by mmdmrishabh**
Pull Request (PR) Request Template:

### Problem

Add atc button with line item prop.

[Monday board link](#)

[Preview  link](#)

(Always do a shopify theme share for attaching a link here.)

### Solution
Added atc used shopify's ajax ap
Screenshot 2024-02-27 at 3 53 26 PM
i.
### Impacted Areas
- PDP 
- PLP

### Done Checklist
- [x] Performed unit testing to ensure functionality and identify potential issues.
- [x] Followed basic code standards and best practices. (indentation, media queries, basic customizer things)
- [ ] Added comments to explain complex logic or improve code readability.
- [ ] Followed Naming conventions and proper variable declaration.
- [ ] Handled edge cases and errors gracefully to prevent crashes.
- [ ] Optimized the code for production to ensure performance.

Please review this PR and provide feedback. Thank you!

### CODE REVIEW

File: sections/main-product.liquid [hunk 1]
```diff
@@ -471,7 +471,7 @@
                   data-quantity = "2",
                   class="product-form__submit button custom-button-secondary_color"
                 >
-                  Add to cart 2
+                  Add to cart 2 - Changes from feature branch
```
Comment (arulmurugan-marmeto) [inline sections/main-product.liquid]: Hey @mmdmrishabh  this is not the correct way to do this thing please change this to `let`

**Review by arulmurugan-marmeto — CHANGES_REQUESTED** [review]
Hey @mmdmrishabh  make changes then we will merge it

File: sections/main-product.liquid (hunk 1 above)
Comment (mmdmrishabh) [reply sections/main-product.liquid]: Hey dude it was not an issue just fixed

**Review by arulmurugan-marmeto — APPROVED** [review]
Hey good job 

Code review score : 4.5 / 5

**Pull request merged**